### PR TITLE
[GOVCMSD8-318] Install config_readonly 8.x-1.0-beta3

### PR DESCRIPTION
**Pull request by suhyeonh**
Install config_readonly 8.x-1.0-beta3

### CODE REVIEW

**Comment by tobybellwood** [comment]
we will need to set this to be active in prod as per https://www.drupal.org/project/config_readonly
at
https://github.com/govCMS/govcms8lagoon/blob/master/.docker/images/govcms8/settings/production.settings.php

File: composer.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
         "drupal/clamav":"1.1.0",
         "drupal/coi": "1.0.0",
         "drupal/config_override_core_fields": "^1.0",
+        "drupal/config_readonly":"^1.0",
```
Comment (fubarhouse) [inline composer.json]: To ensure we're explicitly not going to inherit unexpected changes should we pin to `1.0-beta3` - at least until `1.0` is released. At that point we could move back to `^1.0`, but before that we will inherit any new releases and with it database updates etc.

Happy to get feedback on this, but I generally feel this should be pinned to the specified version in the PR title.

**Comment by AlexSkrypnyk** [comment]
@suhyeonh Could you please resolve conflicts in this PR. Thank you

**Comment by simesy** [comment]
Wait, what? This looks like it will lock all production sites by default?

**Comment by simesy** [comment]
This needs to be closed as current suggested practice to change production and to not deploy via config if there are no developers to do the deployment.

**Comment by AlexSkrypnyk** [comment]
@fubarhouse could you please re-evaluate and close if not going to be accepted. thanks

**Comment by fubarhouse** [comment]
I don't believe we need to go through that process as we've since mitigated the problem in other ways.
I'm going to close this, and we can re-evaluate if the need arises.

**Pull request closed without merging**